teps/0007-conditions-beta.md

```diff
@@ -629,6 +738,317 @@ Provide a special kind of `runAfter` -- `runAfterEvenWhenSkipped` -- that users
 
 However, this would make what happens to the branch opaque to the guarded `Task` because its ordering-dependent `Tasks` could be executed or not be executed.
 
+### Status 
+
+#### Minimal Skipped 
+
+Add `Skipped Tasks` section to the `PipelineRunStatus` that contains a list of the `PipelineTaskName` of the skipped `Tasks`. The `WhenExpressions` that made the `Task` skipped can be found in the `PipelineSpec`, the `Parameter` variables used can be found in the `PipelineSpec` and the `Results` used from previous `Tasks` can be found in the relevant `TaskRun`. It may be involving for users to reverse-engineer to identify why a `Task` was skipped, but gives us the benefit of significantly reducing the `PipelineRunStatus` compared to what we currently have with `Conditions`. 
```

Comment:
v v minor nit: `involving` -> `more involved` or `more work`?

Reply:
updated that, english is hard 😅